### PR TITLE
OpenAPI: processes_api

### PR DESCRIPTION
**Pull request by markxiong0122**

### CODE REVIEW

File: openapi/api.yaml
Comment (DanielRyanSmith) [inline openapi/api.yaml:419]: This field failed when I ran the tests myself, and the tests were actually looking for a string type, rather than a list of strings

Comment (markxiong0122) [reply openapi/api.yaml:419]: I set it to array because the in approval_defs.py it's defined as either string / list[str]. It seems like it won't be a list given the tests we had. I typed them as string and it worked.

Comment (DanielRyanSmith) [reply openapi/api.yaml]: This may be an issue we need to look at before landing. Are there different circumstances where a string is passed over a list of strings? And do we need to facilitate both?

Comment (markxiong0122) [reply openapi/api.yaml]: yes in [approval_def.py](https://github.com/GoogleChrome/chromium-dashboard/blob/dd4a0a4d6b0fe292b51411bb1d84a36da0205857/internals/approval_defs.py#L114), approvers could either be a single email address or a list of such, but it seems like test cases are only looking for a single string

**Review by DanielRyanSmith — APPROVED** [review]

**Pull request merged**
